lago/utils.py

```diff
@@ -533,12 +534,13 @@ def load_virt_stream(virt_fd):
         dict: Loaded virt config
     """
     try:
-        virt_conf = json.load(virt_fd)
+        virt_conf = collections.OrderedDict()
+        virt_conf = json.load(virt_fd, object_pairs_hook=collections.OrderedDict)
     except ValueError:
         virt_fd.seek(0)
         virt_conf = yaml.load(virt_fd)
```

Comment:
Can we make sure that the order is also the same when loaded from yaml? We are using it in more and more places (is nicer to work with manually than json)